bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -2,392 +2,4 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
-{% from "macros.html" import google_play_button with context %}
```

Comment:
What is this template for now that the logic has moved to `notes.html`? I don't see it being used anywhere else.

Reply:
It's supposed to be used for release Firefox, Firefox for Android, and Firefox iOS.

Reply:
bedrock/bedrock/releasenotes/views.py line 31 handles this I think?

Reply:
OOOOOOOOOhhhh got it. Silly me. I knew I was missing something 👍